x-pack/lib/monitoring/monitoring.rb

```diff
@@ -187,7 +187,7 @@ def additionals_settings(settings)
       settings.register(LogStash::Setting::String.new("node.uuid", ""))
     rescue => e
       logger.error e.message
-      logger.error e.backtrace
+      logger.error e.backtrace.to_s
```

Comment:
Just randomly ran into this with the errors fixed, we need a `String` here, a backtrace doesn't work automatically => fixed